src/backend/marsha/urls.py

```diff
@@ -62,6 +63,7 @@
     path("videos/<uuid:uuid>", VideoView.as_view(), name="video_public"),
     path("documents/<uuid:uuid>", DocumentView.as_view(), name="document_public"),
     # API
+    path("api/pairing-challenge", pairing_challenge, name="pairing_challenge"),
```

Comment:
How come there's no url for pairing request?

Reply:
As a secret request is related to a video, the endpoint is `/api/videos/{video.id}/pairing-secret/`

Reply:
True